unit-tests/test-fw-update.py

```diff
@@ -84,13 +90,13 @@ def find_image_or_exit( product_name, image_mask ):
     # For example: for 'Intel RealSense L53X Recovery'
     # Take the 'L53' and try to concatenate it 'X' + postfix and than "XX" + postfix
     # Until find file or exit if not find.
-    x = 'X'
-    r = match.regs[1]
+    x = ''
+    start_index, end_index = match.span(1)
 
-    for i in range(1, 3):
-        pn = product_name[r[0]:r[1]]
+    for i in range(0, end_index-start_index):
+        pn = product_name[start_index:end_index-i]
 
-        image_name = '(^|/)' + product_name[r[0]:r[1]-i] + x + "_FW_Image-" + image_mask + ".bin" + '$'
+        image_name = '(^|/)' + pn + x + "_FW_Image-" + fw_version + ".bin" + '$'
```

Comment:
`.bin` -> r`\.bin'
It's a regex, right?

Reply:
done